tests/Console/Command/AddPrefixCommandTest.php

```diff
@@ -0,0 +1,254 @@
+<?php
+
+/*
+ * This file is part of the webmozart/php-scoper package.
+ *
+ * (c) Bernhard Schussek <bschussek@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Humbug\PhpScoper\Console\Command;
+
+use PHPUnit\Framework\Assert;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\ApplicationTester;
+use Symfony\Component\Filesystem\Filesystem;
+use function Humbug\PhpScoper\makeTempDir;
+
+/**
+ * @covers \Humbug\PhpScoper\Console\Command\AddPrefixCommand
+ */
+class AddPrefixCommandTest extends TestCase
+{
+    const FIXTURES_DIR = __DIR__.DIRECTORY_SEPARATOR.'..'.DIRECTORY_SEPARATOR.'..'.DIRECTORY_SEPARATOR.'../fixtures';
+    
+    /**
+     * @var ApplicationTester
+     */
+    private $appTester;
+
+    /**
+     * @var string
+     */
+    private $cwd;
+
+    /**
+     * @var string
+     */
+    private $tempDir;
+
+    /**
+     * @inheritdoc
+     */
+    protected function setUp()
+    {
+        if (file_exists($this->tempDir)) {
+            chdir($this->tempDir);
+        }
+
+        if (null !== $this->appTester) {
+            return;
+        }
+
+        $this->cwd = getcwd();
+
+        $application = new Application('php-scoper-test');
+        $application->addCommands([
+            new AddPrefixCommand(),
+        ]);
+        $application->setAutoExit(false);
+
+        $this->appTester = new ApplicationTester($application);
+
+
+        $this->tempDir = makeTempDir('php-scoper', __CLASS__);
+
+        $filesystem = new Filesystem();
+        $filesystem->mirror(self::FIXTURES_DIR.DIRECTORY_SEPARATOR.'original', $this->tempDir);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    protected function tearDown()
+    {
+        chdir($this->cwd);
+
+        $filesystem = new Filesystem();
+        $filesystem->remove($this->tempDir);
+    }
+
+    public function test_scope_all_files_found_in_a_directory()
+    {
+        $this->appTester->run(
+            [
+                'add-prefix',
+                'prefix' => 'MyPrefix',
+                'paths' => [
+                    $basePath = self::FIXTURES_DIR.DIRECTORY_SEPARATOR.'original'.DIRECTORY_SEPARATOR.'dir'.DIRECTORY_SEPARATOR.'dir'
+                ],
+            ],
+            [
+                'capture_stderr_separately' => true,
+                'set'
+            ]
+        );
+
+        $expected = <<<EOF
+Scoping $basePath/MyClass.php. . . Success
+Scoping $basePath/MySecondClass.php. . . Success
+Scoping $basePath/MyThirdClass.php. . . Success
+
+EOF;
+        $expected = str_replace('/', DIRECTORY_SEPARATOR, $expected);
+
+        Assert::assertEmpty(
```

Comment:
Because it's a static method so it's supposed to be called that way. Also IIRC at some point it was discussed to no longer have `$this->assert*` in PhpUnit (there was a deprecation). I guess it was reverted.

That said I don't feel strongly about it so I can revert that if looks too weird.